### PR TITLE
:=sosha_evanS2005.@/.comupdated godot (2.0)

### PR DESCRIPTION
**Pull request by earlevans**

### CODE REVIEW

File: Casks/godot.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 cask 'godot' do
-  version '1.1'
```
Comment (earlevans) [inline Casks/godot.rb]: sha256

Comment (earlevans) [inline Casks/godot.rb]: 1.1

**Comment by adidalal** [comment]
https://github.com/caskroom/homebrew-cask/commit/bda8d5993400ecffb7ab6024141282ce41029509

**Comment by vitorgalvao** [comment]
Damn bots. It’s even the same commit. It had already made some nonsense comment early on, in another issue.

**Comment by adidalal** [comment]
Thought so - cleaned up, regardless.

**Pull request closed without merging**